src/fsspec_xrootd/xrootd.py

```diff
@@ -168,10 +169,21 @@ def __init__(
             If true, synchronous methods will not be available in this instance
         loop:
             Bring your own loop (for sync methods)
+        storage_options:
```

Comment:
Here you should add the two storage options explicitly as keyword arguments of the init function.
The idea is that, if you have a hierarchy of classes like
```python
class Base:
    def __init__(self, **kwargs) -> None:
        # hopefully all arguments are handled by now
        assert not kwargs


class Foo(Base):
    def __init__(self, foo_arg: str | None = None, **kwargs) -> None:
        super().__init__(**kwargs)
        self._foo = foo_arg


class Bar(Foo):
    def __init__(self, bar_arg: int | None = None, **kwargs) -> None:
        super().__init__(**kwargs)
        self._bar = bar_arg
```
then the arguments are all accounted for by the time you get to base, in theory. This also it makes it possible to declare the type and default arguments in the initializer signature.
```python
>>> obj = Bar(foo_arg="hi")
>>> print(obj.__dict__)
{'_foo': 'hi', '_bar': None}
```
An even better pattern is to redeclare all arguments so the signature of `Bar()` is accurate. But this is hard to maintain when the class we are inheriting from is in an upstream library.